### DISCUSSION TITLE
Dividing one column by another?

### DISCUSSION THREAD

**Discussion by danielrosehill** [Budibase/budibase#14043] [Help] [answered]
Very simple table with somebody's Twitter follower and following count:

![image](https://github.com/Budibase/budibase/assets/40233203/a92aab77-db02-4c8d-85c2-7da3d36350fc)

Would like to calculate the ratio of followers/following.

Should this do the trick?

![image](https://github.com/Budibase/budibase/assets/40233203/15a77864-86fc-4b73-b231-aca3b4d8ad7f)

**Comment by mike12345567** [comment, marked as answer]
Hi @danielrosehill - generally I would recommend using JS for stuff like this, but HBS helpers like `divide` are used all in one statement e.g.
```
{{ divide Followers Following }}
```
If you hover over the helpers it'll provide an example of how to use them:
![image](https://github.com/Budibase/budibase/assets/4407001/c164e22c-756c-49c3-b304-ca4da8a1b19e)
